tests/foreman/cli/test_discoveredhost.py

```diff
@@ -118,6 +147,28 @@ def test_rhel_pxeless_discovery_provisioning(
         delay=40,
     )
     discovered_host = sat.api.DiscoveredHost().search(query={'mac': mac})[0]
+    if (
+        sat.cli.DiscoveredHost.list(
+            {
+                'organization-id': provisioning_hostgroup.organization[0].id,
+                'location-id': provisioning_hostgroup.location[0].id,
+            }
+        )
+        == []
+    ):
+        with sat.ui_session() as session:
+            session.organization.select(org_name="Any organization")
+            session.location.select(loc_name="Any location")
+            session.discoveredhosts.apply_action(
```

Comment:
Why using a ui_session here in a CLI test?

Reply:
This is the only known workaround for the bug to assign organization and location to the discovered host.